astroquery/gemini/core.py

```diff
@@ -526,4 +527,11 @@ def _gemini_json_to_table(json):
         "release",
         "dec"]
 
+
+@deprecated(since='v0.4.3', alternative='GeminiObservationsClass')
+class ObservationsClass(GeminiObservationsClass):
+    pass
+
+
+GeminiObservations = GeminiObservationsClass()
 Observations = ObservationsClass()
```

Comment:
This line is causing the warning. I guess the brute force way is to silence the warning just for this line...?

Reply:
Yes, the issue is that if I put a filter in ``__init__.py``, then we rightly don't get the warning at module import time. However, then we don't get it at all for any usage of `Observations`, and that's what probably users used the most as the docs is full of `Observations.xzy()`.